tests/rt.sh

```diff
@@ -610,65 +610,80 @@ echo "Machine: " $MACHINE_ID "    Account: " $ACCNR
 
 if [[ $MACHINE_ID = wcoss2 ]]; then
 
-  module load ecflow/5.6.0.13
+  if [[ "${ECFLOW:-false}" == true ]] ; then
+    module load ecflow/5.6.0.13
+  fi
   module load intel/19.1.3.304 python/3.8.6
-  ECFLOW_START=${ECF_ROOT}/scripts/server_check.sh
-  export ECF_OUTPUTDIR=${PATHRT}/ecf_outputdir
-  export ECF_COMDIR=${PATHRT}/ecf_comdir
-  rm -rf ${ECF_OUTPUTDIR} ${ECF_COMDIR}
-  mkdir -p ${ECF_OUTPUTDIR}
-  mkdir -p ${ECF_COMDIR}
+  if [[ "${ECFLOW:-false}" == true ]] ; then
+    ECFLOW_START=${ECF_ROOT}/scripts/server_check.sh
+    export ECF_OUTPUTDIR=${PATHRT}/ecf_outputdir
+    export ECF_COMDIR=${PATHRT}/ecf_comdir
+    rm -rf ${ECF_OUTPUTDIR} ${ECF_COMDIR}
+    mkdir -p ${ECF_OUTPUTDIR}
+    mkdir -p ${ECF_COMDIR}
+  fi
   export colonifnco=":output"  # hack
```

Comment:
This line should be inside 'ecflow' if block. But don't bother with it now if you already started testing. Can be done later.

Reply:
I don't know the ecflow support code well enough to know what is specific to it.

Yes, I've started Rocoto tests on Hera and Hercules.

Reply:
@zach1221 @FernandoAndrade-NOAA we will cross check on other machines.

Reply:
Derecho fails with Rocoto, saying the job cannot be submitted with missing node size information. So, do we need to add derecho node size to rt_utils I assume? Ecflow still works.

Reply:
Did it work in Rocoto mode on Derecho before?

Reply:
It seems the variable is ROCOTO_NODESIZE:

```
${ROCOTO_NODESIZE:+<nodesize>$ROCOTO_NODESIZE</nodesize>}
```

Reply:
> @zach1221 @FernandoAndrade-NOAA we will cross check on other machines.

Just adding verification from my side that there were no issues running a small set of tests with `-r` on Gaea, Jet, Hercules, and Orion.